get/getAllItems/index.js

```diff
@@ -1,21 +1,39 @@
 const AWS = require('aws-sdk');
 
-AWS.config.update( {region: 'us-east-2'} );
+AWS.config.update(
+    {
+        region: 'us-east-2'
+    }
+);
 
 exports.handler = async (event) => {    
     const ddb = new AWS.DynamoDB();    
-    const params = {
+    const params = 
+    {
         "TableName" : "Users"
     };
 
     console.log("Fetching all users from db...");
     let result;
     
-    try {
-        result = (await ddb.scan(params).promise()).Items;
+    try
+    {
+        result ={
+            statusCode: 200,
+            body: (await ddb.scan(params).promise()).Items
+        } 
+        
+        for(let value in result.body){
+            result.body[value] = result.body[value];
+        }
+        result.body = JSON.stringify(result.body);
     }
-    catch(err) {
-        result = err.message;
+    catch(err)
+    {
+        result = {
+            statusCode: 500,
```

Comment:
Change the status code to 400. 500 is internal server error but 400 is content not found or bad input, etc..
```suggestion
            statusCode: 400,
```